### DISCUSSION TITLE
I wonder if there are decrypted assets for downloading,and I would appreciate it very much if anyone could.

### DISCUSSION THREAD

**Discussion by loverorser** [Melledy/Grasscutter#1791] [Q&A] [answered]
Hey guys,I am making a game by Unity based on its assets,but it took me a lot of time to get those undecrypted resources.I've tried many ways,such as AssetStudio,Asset Ripper,UnityPy,UABE and so on.Unfortunately,they all failed.
Now I am looking for those assets : Animation Clips,Particle System resources and character models (like FBX format).
I would appreciate it very very much if anyone can give me such resources or the way to figure it out.

**Comment by Petomai** [comment, marked as answer]
Sounds like an interesting project. The game company occasionally releases their models, people use them for MMD on YouTube. I don't know if they release all of them though. I'm kind of noob and don't even have a server set up yet but I found the following if it helps. [Unity tools ](https://github.com/imadr/Unity-game-hacking)

**Reply by loverorser** [reply]
Thank you very much!That should help me a lot . :)
